src/main/java/org/jpeek/Report.java

```diff
@@ -54,6 +54,14 @@
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
 final class Report {
+    /**
+     * Default mean.
+     */
+    private static final double DEFAULT_MEAN = 0.5d;
```

Comment:
@llorllale These ctor/default param changes were not part of the task, right? Don't forget traceability, a PR should cover exactly what its ticket is about, otherwise we can't trace back changes properly.

Normally, changes out of the scope of the ticket should be removed and a new issue should be filed, but since it's a sandbox project let's leave it like this. Just state in the PR's description that you also did this change.